### DISCUSSION TITLE
Only (slight) color change is used to indicate PR approvals between active and deactivated users

### DISCUSSION THREAD

**Discussion by bgrzesiak-fetch** [community/community#72639] [Accessibility] [answered]
### Select Topic Area

Product Feedback

## Problem
**Given** a PR has requested reviews from other members in an organization
**When** another member approves the PR
**And** that member subsequently gets deactivated from the organization
**Then** the green checkmark becomes gray
**But** the color distinction is extremely slight and probably cannot be discerned by those with some forms of color blindness

I don't have color blindness, and it was hard for me to tell the difference.

## Proposal
Change the gray checkmark that represents "was approved by an organization member who has since been deactivated" to something that doesn't just rely on the gray-ness. Perhaps a dotted-line checkmark?

## Example
![Screenshot 2023-10-26 at 10 23 45 AM](https://github.com/community/community/assets/142844457/7d212546-c232-4651-a7d4-dc035ed5f2d2)

**Comment by ghostinhershell** [comment]
Hi @bgrzesiak-fetch, 

Thanks so much for reporting this and welcome to our community! I'll go ahead and file this with our Accessibility team for review.

**Comment by cehfisher** [comment]
Hello @bgrzesiak-fetch - thank you for providing this feedback! We have filed this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Hello @bgrzesiak-fetch - great news, there have been some updates recently that hopefully have resolved your reported issue! Could you please re-test when you have time?

For more details on the recent changes, please see this [discussion post](https://github.com/orgs/community/discussions/169699) by @ghostinhershell and this [changelog post](https://github.blog/changelog/2025-08-14-clearer-pull-request-reviewer-status-and-enhanced-email-filtering).

Thanks!
